pkg/search/postgres/common.go

```diff
@@ -133,7 +133,11 @@ func (q *query) getPortionBeforeFromClause() string {
 		var primaryKeyPaths []string
 		// Always select the primary keys first.
 		for _, pk := range q.Schema.PrimaryKeys() {
-			primaryKeyPaths = append(primaryKeyPaths, qualifyColumn(pk.Schema.Table, pk.ColumnName))
+			var cast string
+			if pk.SQLType == "uuid" {
```

Comment:
How about re-using `postgres/walker/types.go` here instead of using the string representation?
I haven't looked too deep into it, but I saw that you already import `postgres/walker`, so I would assume that we do not end up in a circular dependency. 
However, if it's not possible, we should create an unexported const for this within this file.